libmachine/mcnflag/flag.go

```diff
@@ -67,5 +67,5 @@ func (f BoolFlag) String() string {
 }
 
 func (f BoolFlag) Default() interface{} {
-	return nil
+	return false
```

Comment:
this seems like it could affect a lot of things, i like the change but are we sure this doesn't break a driver expecting nil?

Reply:
My search found that there is only one place where the `Default` method is called. This place was here (which they immediately switched `nil` to `false`): https://github.com/rancher/machine/blob/master/commands/create.go#L353-L358

Reply:
and i see you updated that. so just removing the check there and returning it properly here.. it's set as returning an interface tho why can't it return bool then?

Reply:
It's implementing an interface: https://github.com/rancher/machine/blob/master/libmachine/mcnflag/flag.go#L5-L8

Reply:
silly